MaxText/pyconfig.py

```diff
@@ -71,6 +67,12 @@ def validate_model_name(s: str) -> bool:
       "Invalid model name was passed. Valid options ", valid_model_names
     )
 
+def validate_overwritten_keys(keys1: list[str], keys2: list[str]):
```

Comment:
validate_no_keys_overwritten_twice would be a crisper name.

Reply:
Sure, I will update that. Thank you!